docs/releases/2.1.txt

```diff
@@ -96,6 +96,12 @@ Minor features
 
 * ...
 
+:mod:`django.core.management`
```

Comment:
you should be using the `Management Commands` header below 😉

Reply:
A release note isn't needed as this is more of a bug fix than a feature.

Reply:
Fair enough. I guess it can't affect third party apps anyway since they can't be loaded if `INSTALLED_APPS` isn't there for them to be in.

Reply:
so should i hard reset it to the last commit and push it again?

Reply:
yeah

we actually merge only single commits so if you could squash the result into one with the same title as the PR that would be great

Reply:
@adamchainz If that's your policy, you should consider enforcing [squash and merge](https://github.com/blog/2141-squash-your-commits) in your repo. That way external contributors don't need to do work to squash and force push to the PR branch.

Reply:
@adamchainz  I squashed everying into one commit. hope it works now. 😄 😄

Reply:
@tobiasvl sometimes we merge PR's with two commits, and anyway there's a particular commit format for Django that GH's "Squash and Merge" doesn't really help with.